ovs/lib/storagerouter.py

```diff
@@ -767,8 +775,8 @@ def add_vpool(parameters):
         storagedriver_config.configure_content_addressed_cache(serialize_read_cache=False,
                                                                read_cache_serialization_path=[])
         storagedriver_config.configure_scocache(scocache_mount_points=writecaches,
-                                                trigger_gap='1GB',
-                                                backoff_gap='2GB')
+                                                trigger_gap=Toolbox.convert_to_human_readable(size=smallest_write_partition / 2),
```

Comment:
Yes, this is expected. According to Arne the volumedriver accepts both GB and GiB values and since we use the 1024 based values everywhere, i've changed it here too